packages/trader/src/App/Components/Elements/PositionsDrawer/positions-drawer.tsx

```diff
@@ -120,11 +149,13 @@ const PositionsDrawer = observer(({ ...props }) => {
                     server_time={server_time}
                     getContractById={getContractById}
                     is_mobile={is_mobile}
+                    is_positions_drawer_on={is_positions_drawer_on}
                     current_focus={current_focus}
                     removeToast={removeToast}
                     setCurrentFocus={setCurrentFocus}
                     should_show_cancellation_warning={should_show_cancellation_warning}
                     toggleCancellationWarning={toggleCancellationWarning}
+                    togglePositionsDrawer={toggleDrawer}
```

Comment:
@henry-deriv why do we need 2 extra props here (is_positions_drawer_on, togglePositionsDrawer) ? :)  
They are not used in PositionsDrawerCardItem or PositionsDrawerCard as I can see.

Reply:
U r right. It was just there from before but i removed them.